File: python/program/sport.py
from mobil import Mobil

class Sport(Mobil):
    def __init__(self, nama, merk, mesin, seat, warna, tahun, harga, horsepower, topspeed, turbo):
        super().__init__(nama, merk, mesin, seat, warna, tahun, harga)
        self.horsepower = horsepower
        self.topspeed = topspeed
        self.turbo = turbo

    def getHorsepower(self):
        return self.horsepower
    
    def getTopSpeed(self):
        return self.topspeed

    def getTurbo(self):
        return self.turbo
    
    def setHorsepower(self, horsepower):
        self.horsepower = horsepower
    
    def setTopSpeed(self, topspeed):
        self.topspeed = topspeed
    
    def setTurbo(self, turbo):
        self.turbo = turbo

    def tampilkan_info(self):
        """Method override untuk menampilkan informasi mobil sport"""
        super().tampilkan_info()
        print(f"Horsepower: {self.horsepower} HP")
        print(f"Top Speed : {self.topspeed} km/h")
        print(f"Turbo     : {'Ya' if self.turbo else 'Tidak'}")

File: java/program/Sport.java
class Sport extends Mobil{
    private String horsepower;
    private String topspeed;
    private boolean turbo;

    public Sport() {
        super();
        this.horsepower = "";
        this.topspeed = "";
        this.turbo = false;
    }

    public Sport(String nama, String merk, Mesin mesin, int seat, String warna, int tahun, String harga, String horsepower, String topspeed, boolean turbo) {
        super(nama, merk, mesin, seat, warna, tahun, harga);
        this.horsepower = horsepower;
        this.topspeed = topspeed;
        this.turbo = turbo;
    }

    public String getHorsepower() {
        return horsepower;
    }

    public String getTopspeed() {
        return topspeed;
    }

    public boolean isTurbo() {
        return turbo;
    }

    public void setHorsepower(String horsepower) {
        this.horsepower = horsepower;
    }

    public void setTopspeed(String topspeed) {
        this.topspeed = topspeed;
    }

    public void setTurbo(boolean turbo) {
        this.turbo = turbo;
    }

    @Override
    public void tampilkanInfo() {
        super.tampilkanInfo();
        System.out.println("Horsepower: " + horsepower);
        System.out.println("Top Speed: " + topspeed);
        System.out.println("Turbo: " + (turbo ? "Yes" : "No"));
    }
}